### ISSUE TITLE
Unsupported Kmer value defaults to A

### ISSUE THREAD

**Issue by ghuls** [seq-lang/seq#152]
All unsupported Kmer values default to A. IUPAC also does not seem to be supported (which is fine as long as it does not convert them implicitly to A).

```
# IUPAC
#A      Adenine
#C      Cytosine
#G      Guanine
#T (or U)       Thymine (or Uracil)
#R      A or G
#Y      C or T
#S      G or C
#W      A or T
#K      G or T
#M      A or C
#B      C or G or T
#D      A or G or T
#H      A or C or T
#V      A or C or G
#N      any base
#. or -         gap

for kmer in (k'A', k'C', k'G', k'T', k'R', k'Y', k'S', k'W', k'K', k'M', k'B', k'D', k'H', k'V', k'N', k'_'):
    print(kmer, abs(kmer - k'A'), abs(kmer - k'C'), abs(kmer - k'G'), abs(kmer - k'T'))
```

Output:
```
(A, 0, 1, 1, 1)
(C, 1, 0, 1, 1)
(G, 1, 1, 0, 1)
(T, 1, 1, 1, 0)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
(A, 0, 1, 1, 1)
```

**Comment by arshajii** [comment]
Yes, this is known behavior. In the next major release there will be an exception raised when trying to create a k-mer with non-ACGT bases. Thanks for the report, though! Note that if you use the `kmers` or `kmers_with_pos` functions then non-ACGT bases will be skipped automatically during the k-merization.

**Comment by inumanag** [comment]
Resolved in #132  (throws an exception now).
